result/nd.yaml

```diff
@@ -0,0 +1,73 @@
+openapi: 3.1.0
+
+info:
+  title: Neighbor Discovery table models
+  version: ^0.0.0
+
+components:
+  schemas:
+    Nd.Entries.Request:
+      description: >-
+        The request to retrieve Neighbor Discovery entries per IPv6 interface.
+      type: object
+      properties:
+        ipv6_names:
+          description: >-
+            The names of IPv6 interfaces for which ND entries will be retrieved.
+            If no interface names are specified then the result set will contain all ND entries.
+          type: array
+          items:
+            type: string
+          x-constraint:
+            - "/components/schemas/Device.Ipv6/properties/name"
+        column_names:
+          description: >-
+            The list of column names that the returned result set will contain.
+            If the list is empty then all columns will be returned.
+            The column ipv6_name will always be present in the returned result set.
+          type: array
+          items:
+            type: string
+            enum:
+              - ipv6_name
+              - neighbor_ip
+              - link_layer_address
+
+    Nd.Entries.Response:
+      description: >-
+        A container for Neighbor Discovery entries.
+      type: object
+      properties:
+        resolved:
+          description: >-
+            Contains the Neighbor Discovery entries where link-layer address of the neighbor is already known.
```

Comment:
Contains the Neighbor Discovery entries where link-layer address (MAC) of the neighbor is already known.